### ISSUE TITLE
packer crash while building (with atlas configuration)

### ISSUE THREAD

**Issue by unitsche** [mitchellh/packer#1754]
2014/12/12 20:29:41 [INFO] Packer version: 0.7.5  9cd66feeacbd9cb318b72eb5ed59428c5b8c37ac+CHANGES
2014/12/12 20:29:41 Packer Target OS/Arch: darwin amd64
2014/12/12 20:29:41 Built with Go Version: go1.3.3
2014/12/12 20:29:41 [DEBUG] Discoverd plugin: amazon-chroot = /usr/local/bin/packer-builder-amazon-chroot
2014/12/12 20:29:41 [DEBUG] Discoverd plugin: amazon-ebs = /usr/local/bin/packer-builder-amazon-ebs
2014/12/12 20:29:41 [DEBUG] Discoverd plugin: amazon-instance = /usr/local/bin/packer-builder-amazon-instance
2014/12/12 20:29:41 [DEBUG] Discoverd plugin: digitalocean = /usr/local/bin/packer-builder-digitalocean
2014/12/12 20:29:41 [DEBUG] Discoverd plugin: docker = /usr/local/bin/packer-builder-docker
2014/12/12 20:29:41 [DEBUG] Discoverd plugin: googlecompute = /usr/local/bin/packer-builder-googlecompute
2014/12/12 20:29:41 [DEBUG] Discoverd plugin: null = /usr/local/bin/packer-builder-null
2014/12/12 20:29:41 [DEBUG] Discoverd plugin: openstack = /usr/local/bin/packer-builder-openstack
2014/12/12 20:29:41 [DEBUG] Discoverd plugin: parallels-iso = /usr/local/bin/packer-builder-parallels-iso
2014/12/12 20:29:41 [DEBUG] Discoverd plugin: parallels-pvm = /usr/local/bin/packer-builder-parallels-pvm
2014/12/12 20:29:41 [DEBUG] Discoverd plugin: qemu = /usr/local/bin/packer-builder-qemu
2014/12/12 20:29:41 [DEBUG] Discoverd plugin: virtualbox-iso = /usr/local/bin/packer-builder-virtualbox-iso
2014/12/12 20:29:41 [DEBUG] Discoverd plugin: virtualbox-ovf = /usr/local/bin/packer-builder-virtualbox-ovf
2014/12/12 20:29:41 [DEBUG] Discoverd plugin: vmware-iso = /usr/local/bin/packer-builder-vmware-iso
2014/12/12 20:29:41 [DEBUG] Discoverd plugin: vmware-vmx = /usr/local/bin/packer-builder-vmware-vmx
2014/12/12 20:29:41 [DEBUG] Discoverd plugin: atlas = /usr/local/bin/packer-post-processor-atlas
2014/12/12 20:29:41 [DEBUG] Discoverd plugin: compress = /usr/local/bin/packer-post-processor-compress
2014/12/12 20:29:41 [DEBUG] Discoverd plugin: docker-import = /usr/local/bin/packer-post-processor-docker-import
2014/12/12 20:29:41 [DEBUG] Discoverd plugin: docker-push = /usr/local/bin/packer-post-processor-docker-push
2014/12/12 20:29:41 [DEBUG] Discoverd plugin: docker-save = /usr/local/bin/packer-post-processor-docker-save
2014/12/12 20:29:41 [DEBUG] Discoverd plugin: docker-tag = /usr/local/bin/packer-post-processor-docker-tag
2014/12/12 20:29:41 [DEBUG] Discoverd plugin: vagrant = /usr/local/bin/packer-post-processor-vagrant
2014/12/12 20:29:41 [DEBUG] Discoverd plugin: vagrant-cloud = /usr/local/bin/packer-post-processor-vagrant-cloud
2014/12/12 20:29:41 [DEBUG] Discoverd plugin: vsphere = /usr/local/bin/packer-post-processor-vsphere
2014/12/12 20:29:41 [DEBUG] Discoverd plugin: ansible-local = /usr/local/bin/packer-provisioner-ansible-local
2014/12/12 20:29:41 [DEBUG] Discoverd plugin: chef-client = /usr/local/bin/packer-provisioner-chef-client
"crash.log" [readonly] 5117L, 589060C
2014/12/12 20:50:51 packer-post-processor-atlas:        /Users/mitchellh/code/go/src/github.com/hashicorp/go-msgpack/codec/decode.go:630 +0x7c
2014/12/12 20:50:51 packer-post-processor-atlas: github.com/hashicorp/go-msgpack/codec.(*rpcCodec).read(0xc2081f60c0, 0x2fdfe0, 0xc20827c060, 0x0, 0x0)
2014/12/12 20:50:51 packer-post-processor-atlas:        /Users/mitchellh/code/go/src/github.com/hashicorp/go-msgpack/codec/rpc.go:88 +0xd6
2014/12/12 20:50:51 packer-post-processor-atlas: github.com/hashicorp/go-msgpack/codec.(*goRpcCodec).ReadResponseHeader(0xc2081f60c0, 0xc20827c060, 0x0, 0x0)
2014/12/12 20:50:51 packer-post-processor-atlas:        /Users/mitchellh/code/go/src/github.com/hashicorp/go-msgpack/codec/rpc.go:123 +0x5e
2014/12/12 20:50:51 packer-post-processor-atlas: net/rpc.(*Client).input(0xc2083223c0)
2014/12/12 20:50:51 packer-post-processor-atlas:        /Users/mitchellh/code/3rdparty/go/src/pkg/net/rpc/client.go:109 +0xda
2014/12/12 20:50:51 packer-post-processor-atlas: created by net/rpc.NewClientWithCodec
2014/12/12 20:50:51 packer-post-processor-atlas:        /Users/mitchellh/code/3rdparty/go/src/pkg/net/rpc/client.go:201 +0x8b
2014/12/12 20:50:51 Deleting original artifact for build 'virtualbox-iso'
2014/12/12 20:50:51 /usr/local/bin/packer-post-processor-atlas: plugin process exited
2014/12/12 20:50:51 ui error: ^[[1;31mBuild 'virtualbox-iso' errored: 1 error(s) occurred:

* Post-processor failed: unexpected EOF^[[0m
2014/12/12 20:50:51 Builds completed. Waiting on interrupt barrier...
2014/12/12 20:50:51 machine readable: error-count []string{"1"}
2014/12/12 20:50:51 ui error:
==> Some builds didn't complete successfully and had errors:
2014/12/12 20:50:51 machine readable: virtualbox-iso,error []string{"1 error(s) occurred:\n\n* Post-processor failed: unexpected EOF"}
2014/12/12 20:50:51 ui error: --> virtualbox-iso: 1 error(s) occurred:

* Post-processor failed: unexpected EOF
2014/12/12 20:50:51 ui:
==> Builds finished but no artifacts were created.
2014/12/12 20:50:51 waiting for all plugin processes to complete...
2014/12/12 20:50:51 /usr/local/bin/packer-provisioner-shell: plugin process exited
2014/12/12 20:50:51 /usr/local/bin/packer-builder-virtualbox-iso: plugin process exited
~
2014/12/12 20:50:51 packer-post-processor-atlas: github.com/hashicorp/go-msgpack/codec.(*Decoder).decode(0xc208322360, 0x2fdfe0, 0xc20827c060)
2014/12/12 20:50:51 packer-post-processor-atlas:        /Users/mitchellh/code/go/src/github.com/hashicorp/go-msgpack/codec/decode.go:635 +0x58
2014/12/12 20:50:51 packer-post-processor-atlas: github.com/hashicorp/go-msgpack/codec.(*Decoder).Decode(0xc208322360, 0x2fdfe0, 0xc20827c060, 0x0, 0x0)
2014/12/12 20:50:51 packer-post-processor-atlas:        /Users/mitchellh/code/go/src/github.com/hashicorp/go-msgpack/codec/decode.go:630 +0x7c
2014/12/12 20:50:51 packer-post-processor-atlas: github.com/hashicorp/go-msgpack/codec.(*rpcCodec).read(0xc2081f60c0, 0x2fdfe0, 0xc20827c060, 0x0, 0x0)
2014/12/12 20:50:51 packer-post-processor-atlas:        /Users/mitchellh/code/go/src/github.com/hashicorp/go-msgpack/codec/rpc.go:88 +0xd6
2014/12/12 20:50:51 packer-post-processor-atlas: github.com/hashicorp/go-msgpack/codec.(*goRpcCodec).ReadResponseHeader(0xc2081f60c0, 0xc20827c060, 0x0,,
 0x0)
2014/12/12 20:50:51 packer-post-processor-atlas:        /Users/mitchellh/code/go/src/github.com/hashicorp/go-msgpack/codec/rpc.go:123 +0x5e
2014/12/12 20:50:51 packer-post-processor-atlas: net/rpc.(*Client).input(0xc2083223c0)
2014/12/12 20:50:51 packer-post-processor-atlas:        /Users/mitchellh/code/3rdparty/go/src/pkg/net/rpc/client.go:109 +0xda
2014/12/12 20:50:51 packer-post-processor-atlas: created by net/rpc.NewClientWithCodec
2014/12/12 20:50:51 packer-post-processor-atlas:        /Users/mitchellh/code/3rdparty/go/src/pkg/net/rpc/client.go:201 +0x8b
2014/12/12 20:50:51 Deleting original artifact for build 'virtualbox-iso'
2014/12/12 20:50:51 /usr/local/bin/packer-post-processor-atlas: plugin process exited
2014/12/12 20:50:51 ui error: ^[[1;31mBuild 'virtualbox-iso' errored: 1 error(s) occurred:

* Post-processor failed: unexpected EOF^[[0m
2014/12/12 20:50:51 Builds completed. Waiting on interrupt barrier...
2014/12/12 20:50:51 machine readable: error-count []string{"1"}
2014/12/12 20:50:51 ui error:
==> Some builds didn't complete successfully and had errors:
2014/12/12 20:50:51 machine readable: virtualbox-iso,error []string{"1 error(s) occurred:\n\n* Post-processor failed: unexpected EOF"}
2014/12/12 20:50:51 ui error: --> virtualbox-iso: 1 error(s) occurred:

* Post-processor failed: unexpected EOF
2014/12/12 20:50:51 ui:
==> Builds finished but no artifacts were created.
2014/12/12 20:50:51 waiting for all plugin processes to complete...
2014/12/12 20:50:51 /usr/local/bin/packer-provisioner-shell: plugin process exited
2014/12/12 20:50:51 /usr/local/bin/packer-builder-virtualbox-iso: plugin process exited
~
~
2014/12/12 20:50:51 packer-post-processor-atlas: bufio.(*Reader).ReadByte(0xc208322300, 0xc2082ec140, 0x0, 0x0)
2014/12/12 20:50:51 packer-post-processor-atlas:        /Users/mitchellh/code/3rdparty/go/src/pkg/bufio/bufio.go:199 +0x7e
2014/12/12 20:50:51 packer-post-processor-atlas: github.com/hashicorp/go-msgpack/codec.(*ioDecReader).readn1(0xc20822e210, 0x2f4980)
2014/12/12 20:50:51 packer-post-processor-atlas:        /Users/mitchellh/code/go/src/github.com/hashicorp/go-msgpack/codec/decode.go:90 +0x44
2014/12/12 20:50:51 packer-post-processor-atlas: github.com/hashicorp/go-msgpack/codec.(*msgpackDecDriver).initReadNext(0xc2082ec140)
2014/12/12 20:50:51 packer-post-processor-atlas:        /Users/mitchellh/code/go/src/github.com/hashicorp/go-msgpack/codec/msgpack.go:540 +0x48
2014/12/12 20:50:51 packer-post-processor-atlas: github.com/hashicorp/go-msgpack/codec.(*Decoder).decode(0xc208322360, 0x2fdfe0, 0xc20827c060)
2014/12/12 20:50:51 packer-post-processor-atlas:        /Users/mitchellh/code/go/src/github.com/hashicorp/go-msgpack/codec/decode.go:635 +0x58
2014/12/12 20:50:51 packer-post-processor-atlas: github.com/hashicorp/go-msgpack/codec.(*Decoder).Decode(0xc208322360, 0x2fdfe0, 0xc20827c060, 0x0, 0x0)
2014/12/12 20:50:51 packer-post-processor-atlas:        /Users/mitchellh/code/go/src/github.com/hashicorp/go-msgpack/codec/decode.go:630 +0x7c
2014/12/12 20:50:51 packer-post-processor-atlas: github.com/hashicorp/go-msgpack/codec.(*rpcCodec).read(0xc2081f60c0, 0x2fdfe0, 0xc20827c060, 0x0, 0x0)
2014/12/12 20:50:51 packer-post-processor-atlas:        /Users/mitchellh/code/go/src/github.com/hashicorp/go-msgpack/codec/rpc.go:88 +0xd6
2014/12/12 20:50:51 packer-post-processor-atlas: github.com/hashicorp/go-msgpack/codec.(*goRpcCodec).ReadResponseHeader(0xc2081f60c0, 0xc20827c060, 0x0, 0x0)
2014/12/12 20:50:51 packer-post-processor-atlas:        /Users/mitchellh/code/go/src/github.com/hashicorp/go-msgpack/codec/rpc.go:123 +0x5e
2014/12/12 20:50:51 packer-post-processor-atlas: net/rpc.(*Client).in
2014/12/12 20:50:51 packer-post-processor-atlas: bufio.(*Reader).fill(0xc208322300)
2014/12/12 20:50:51 packer-post-processor-atlas:        /Users/mitchellh/code/3rdparty/go/src/pkg/bufio/bufio.go:97 +0x1b3
2014/12/12 20:50:51 packer-post-processor-atlas: bufio.(*Reader).ReadByte(0xc208322300, 0xc2082ec140, 0x0, 0x0)
2014/12/12 20:50:51 packer-post-processor-atlas:        /Users/mitchellh/code/3rdparty/go/src/pkg/bufio/bufio.go:199 +0x7e
2014/12/12 20:50:51 packer-post-processor-atlas: github.com/hashicorp/go-msgpack/codec.(*ioDecReader).readn1(0xc20822e210, 0x2f4980)
2014/12/12 20:50:51 packer-post-processor-atlas:        /Users/mitchellh/code/go/src/github.com/hashicorp/go-msgpack/codec/decode.go:90 +0x44
2014/12/12 20:50:51 packer-post-processor-atlas: github.com/hashicorp/go-msgpack/codec.(*msgpackDecDriver).initReadNext(0xc2082ec140)
2014/12/12 20:50:51 packer-post-processor-atlas:        /Users/mitchellh/code/go/src/github.com/hashicorp/go-msgpack/codec/msgpack.go:540 +0x48
2014/12/12 20:50:51 packer-post-processor-atlas: github.com/hashicorp/go-msgpack/codec.(*Decoder).decode(0xc208322360, 0x2fdfe0, 0xc20827c060)
2014/12/12 20:50:51 packer-post-processor-atlas:        /Users/mitchellh/code/go/src/github.com/hashicorp/go-msgpack/codec/decode.go:635 +0x58
2014/12/12 20:50:51 packer-post-processor-atlas: github.com/hashicorp/go-msgpack/codec.(*Decoder).Decode(0xc208322360, 0x2fdfe0, 0xc20827c060, 0x0, 0x0)
2014/12/12 20:50:51 packer-post-processor-atlas:        /Users/mitchellh/code/go/src/github.com/hashicorp/go-msgpack/codec/decode.go:630 +0x7c
2014/12/12 20:50:51 packer-post-processor-atlas: github.com/hashicorp/go-msgpack/codec.(*rpcCodec).read(0xc2081f60c0, 0x2fdfe0, 0xc20827c060, 0x0, 0x0)
2014/12/12 20:50:51 packer-post-processor-atlas:        /Users/mitchellh/code/go/src/github.com/hashicorp/go-msgpack/codec/rpc.go:88 +0xd6
2014/12/12 20:50:51 packer-post-processor-atlas: github.com/hashicorp/go-msgpack/codec.(*goRpcCodec).ReadResponseHeader(0xc2081f60c0, 0xc20827c060, 0x0, 0x0)
2014/12/12 20:50:51 packer-post-processor-atlas:        /Users/mitchellh/code/go/src/github.com/hashicorp/go-msgpack/codec/rpc.go:123 +0x5e
2014/12/12 20:50:51 packer-post-processor-atlas: net/rpc.(*Client).input(0xc2083223c0)
2014/12/12 20:50:51 packer-post-processor-atlas:        /Users/mitchellh/code/3rdparty/go/src/pkg/net/rpc/client.go:109 +0xda
2014/12/12 20:50:51 packer-post-processor-atlas: created by net/rpc.NewClientWithCodec
2014/12/12 20:50:51 packer-post-processor-atlas:        /Users/mitchellh/code/3rdparty/go/src/pkg/net/rpc/client.go:201 +0x8b
2014/12/12 20:50:51 Deleting original artifact for build 'virtualbox-iso'
2014/12/12 20:50:51 /usr/local/bin/packer-post-processor-atlas: plugin process exited
2014/12/12 20:50:51 ui error: ^[[1;31mBuild 'virtualbox-iso' errored: 1 error(s) occurred:

* Post-processor failed: unexpected EOF^[[0m
2014/12/12 20:50:51 Builds completed. Waiting on interrupt barrier...
2014/12/12 20:50:51 machine readable: error-count []string{"1"}
2014/12/12 20:50:51 ui error:
==> Some builds didn't complete successfully and had errors:
2014/12/12 20:50:51 machine readable: virtualbox-iso,error []string{"1 error(s) occurred:\n\n* Post-processor failed: unexpected EOF"}
2014/12/12 20:50:51 ui error: --> virtualbox-iso: 1 error(s) occurred:

* Post-processor failed: unexpected EOF
2014/12/12 20:50:51 ui:
==> Builds finished but no artifacts were created.
2014/12/12 20:50:51 waiting for all plugin processes to complete...
2014/12/12 20:50:51 /usr/local/bin/packer-provisioner-shell: plugin process exited
2014/12/12 20:50:51 /usr/local/bin/packer-builder-virtualbox-iso: plugin process exited

**Comment by Cbeck527** [comment]
Were you able to solve this? I'm seeing the same issue!

**Comment by sethvargo** [comment]
This is a duplicate of https://github.com/mitchellh/packer/issues/1815. Even though this is "older", let's track #1815 please :smile:.
